### PR TITLE
Asyn form movements

### PR DESCRIPTION
**Pull request by Nany-fork**
This PR will have
- Add modal to make a new category in the form of movement
- Select box using Ajax Jquery (I still don't get this one)

### CODE REVIEW

**Comment by Nany-fork** [comment]
image
image

Se me paso boton mi boton, lo corrijo

**Comment by Nany-fork** [comment]
image
image

**Review by itsmeurbi — APPROVED** [review]
Seems to be legit!
Well done 👍

File: app/views/movement_categories/new.html.erb [hunk 1]
```diff
@@ -1,12 +1,9 @@
-<%= render "navbar" %>
-<br/>
-<div class="form-content">
-  <div class="form-child">
-    <h1>New movement category</h1>
-    <%= render "form", movement_category: @movement_category %>
-    <br>
-    <div>
-      <%= link_to "Back to movement categories", new_movement_path, class: "pure-button pure-button-active" %>
+<%= turbo_frame_tag "modal" do %>
```
Comment (3zcurdia) [inline app/views/movement_categories/new.html.erb:1]: Creo que el turbo frame no es necesario

**Review by 3zcurdia — CHANGES_REQUESTED** [review]
Antes de usar el stack de hotwired déjalo con un stack de rails básico

**Comment by Nany-fork** [comment]
https://user-images.githubusercontent.com/95006480/153100775-9e8900ff-efd2-465a-b6be-328003c9f19c.mp4

File: app/views/movement_categories/new.html.erb (hunk 1 above)
Comment (Nany-fork) [reply app/views/movement_categories/new.html.erb]: Ya lo borre :3

**Review by itsmeurbi — APPROVED** [review]
Seems to be legit! 😃

**Pull request merged**